.copywrite.hcl

```diff
@@ -6,7 +6,7 @@ project {
 
   header_ignore = [
     # changie tooling configuration and CHANGELOG entries (prose)
-    ".changes/unreleased/*.yaml",
+    ".changes/unreleased/**",
```

Comment:
This was a little too aggressive for our changelogs, loosened 😆 
https://github.com/hashicorp/terraform-plugin-go/actions/runs/13858526605/job/38780895022?pr=490